pdf-service/src/utils/commons.js

```diff
@@ -1,4 +1,4 @@
-import { httpRequest } from "../api/api";
+const fetch = require("node-fetch");
```

Comment:
Use `axios`

Reply:
removed unnecessary dependency and use axios for https request call